dev-docs/HOW_THINGS_WORK.md

```diff
@@ -317,3 +317,35 @@ This is pipeline is automatically triggered every night to make sure that the ma
 This pipeline is for end to end testing and creates different builds integrating with other projects of chef.
 These builds are packaged to replicate the different environment in which chef-server will be used by the customers.
 This pipeline is run on a nightly basis using the latest build from the current omnibus pipeline.
+
+## Using S3 instead of Bookshelf
+
+If you would like to configure chef-server to use S3 instead of Bookshelf, you can
```

Comment:
Suggest: 
To configure chef-server to use S3 instead of Bookshelf, enter the following command into the bash shell of your chef-server.  
This example assumes you are using a root login.  
Make obvious substitutions as needed, for example, to `s3_bucket`, `access_key_id`:

It might be worth mentioning why someone might need S3 instead of Bookshelf, but maybe that's obvious internally to the team